### PR TITLE
Add page about updating Rack and CLI in the docs.

### PR DESCRIPTION
**Pull request by soulshake**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by mwarkentin** [comment]
@soulshake Worth adding a note about "required releases"? What they are and why they're needed?

**Comment by soulshake** [comment]
[Add update doc to Management section](https://trello.com/c/jVkZUyIb/756-add-update-doc-to-management-section)

File: _management/updating-rack-and-cli.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Updating a Rack
+
+There are several ways to update a Rack.
+
+### Via the console
+
+After logging into [the Convox web console](https://console.convox.com/), you can see your list of Racks under the `Racks` tab:
+
+![List of Racks]({{ site.url }}/assets/images/docs/what-is-a-rack/list-of-racks.png)
```
Comment (ianmalott) [inline _management/updating-rack-and-cli.md]: I think you can/should drop the `{{ site.url }}`. The image didn't show up for me until I changed the line to...

```
![List of Racks](/assets/images/docs/what-is-a-rack/list-of-racks.png)
```

screen shot 2016-11-14 at 4 31 42 pm

Comment (soulshake) [inline _management/updating-rack-and-cli.md]: Fixed, thanks

File: _management/updating-rack-and-cli.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+$ convox doctor
+### Setup
+[✓] Convox CLI version    
+[snip]
+```
+
+
+## Updating a Rack
+
+There are several ways to update a Rack.
```
Comment (soulshake) [inline _management/updating-rack-and-cli.md]: Replaced with "two"

**Comment by soulshake** [comment]
P.S. As @mwarkentin pointed out, would be good to add a note about required releases soon.

**Pull request merged**